apps/web/src/state/lists/updater.ts

```diff
@@ -48,7 +48,7 @@ export default function Updater(): null {
   })
 
   useSWRImmutable(
-    includeListUpdater && isReady && listState !== initialState ? ['token-list'] : null,
+    includeListUpdater && isReady && listState !== initialState && Object.keys(lists).length ? ['token-list'] : null,
```

Comment:
can we take a look at the reason why `isReady` is `true` but there's no list provided?

Reply:
`lists` may still empty Object, when `isReady` turns true

Reply:
yeah but that's not intended I think. `isReady` should be `true` only when the `lists` are ready, no?